src/entities/activity/ui/items/Date.tsx

```diff
@@ -15,5 +19,11 @@ export const DateItem = ({ value, onValueChange }: Props) => {
     return onValueChange([value])
   }
 
-  return <DateItemBase value={value} onChange={onHandleChange} />
+  return (
+    <Box display="flex" justifyContent="center">
+      <Box sx={{ width }}>
```

Comment:
The width of the component should defined and controlled outside of the component

Reply:
Done